modules/monitoring-understanding-monitoring-stack-in-ha-clusters.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assembly:
+//
+// * observability/monitoring/monitoring-overview.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="understanding-monitoring-stack-in-ha-clusters_{context}"]
+= Understanding the monitoring stack in high-availability clusters
+
+By default, in multi-node clusters,
+ifndef::openshift-dedicated,openshift-rosa[]
+both the default monitoring stack components and 
+endif::openshift-dedicated,openshift-rosa[]
+user-defined monitoring components run in high-availability (HA) mode. This is to prevent data loss and service interruption. 
+
+The component is replicated across two pods, each running on a separate node. This means that the monitoring stack can tolerate the loss of one pod.
```

Comment:
HA is enabled for Prometheus, Alertmanager, Thanos Querier, Thanos Ruler, Metrics server and monitoring plugin.
Exporters aren't HA right now:
* node_exporter can't really be made HA since there's 1 pod running per node.
* Other exporters like kube-state-metrics and openshift-state-metrics could be made HA but there's little incentive from our side.

As far as I can tell, the documentation focuses on stateful components, maybe it's worth mentioning?

Reply:
and metrics-server is since 4.16, before 4.16, it's prometheus-adapter